infrastructure/index.ts

```diff
@@ -59,8 +59,10 @@ const config = {
     marketingPortalStack: stackConfig.get("marketingPortalStack"),
 };
 
-const aiAppStack = new pulumi.StackReference('pulumi/pulumi-ai-app-infra/prod');
+// const aiAppStack = new pulumi.StackReference('pulumi/pulumi-ai-app-infra/prod');
+const aiAppStack = new pulumi.StackReference('pulumi/pulumi-ai-app-infra/pr-pulumi-pulumi.ai-1297');
```

Comment:
can we make the stack reference itself a config value that is specified in the stack config, instead of hard coding it, since it appears to be environment specific?

unless we only have one environment for the ai stack that all docs stacks will reference then maybe this is ok

Reply:
We'll only have a one env really. Its a bit hard to align all the CORS etc to get this going on the test previews for now.

But I still like the idea a lot. Will do a follow up PR.